zhaquirks/const.py

```diff
@@ -118,6 +118,7 @@
 UNKNOWN = "Unknown"
 VALUE = "value"
 ZHA_SEND_EVENT = "zha_send_event"
-ZONE_STATE = 0
+ZONE_STATUS_CHANGE_COMMAND = 0x0000
```

Comment:
Maybe `0x00`?
https://github.com/zigpy/zigpy/blob/1f81b162f84328eb96db18add3104d889fa6c923/zigpy/zcl/clusters/security.py#L104

Reply:
Other constants are also four digits:
```python
ZONE_STATE = 0x0000
ZONE_TYPE = 0x0001
ZONE_STATUS = 0x0002
```
Doesn't really matter. We should migrate everything to grab the IDs from `zigpy` though probably.

Reply:
These others are cluster attributes that are defined this way (4 digits):
* https://github.com/zigpy/zigpy/blob/1f81b162f84328eb96db18add3104d889fa6c923/zigpy/zcl/clusters/security.py#L69-L75

While cluster commands are 2 digits.

Reply:
And yes, it really doesn't matter. (sorry to be picky)